src/test/java/uk/gov/ons/ctp/response/collection/exercise/validation/ValidateSampleUnitsTest.java

```diff
@@ -133,6 +139,100 @@ public void testValidateSampleUnits() throws CTPException {
         .transitionCollectionExercise(collectionExercise, CollectionExerciseEvent.VALIDATE);
   }
 
+  @Test
+  public void testValidateSampleUnitsMultipleCollectionInstruments() throws CTPException {
+    // Given
```

Comment:
This would benefit  from a description showing why we need this tests , and how this test satisfies that . I think such a description would be too long to to fit within a modified function name .

Reply:
added comment